frontend/src/app/main/data/workspace/common.cljs

```diff
@@ -299,10 +302,11 @@
     ptk/UpdateEvent
     (update [_ state]
       ;; We commit the old transaction before starting the new one
-      (-> state
-          (add-undo-entry (get-in state [:workspace-undo :transaction]))
-          (assoc-in [:workspace-undo :transaction] {:undo-changes []
-                                                    :redo-changes []})))))
+      (let [empty-tx {:undo-changes [] :redo-changes []}
```

Comment:
MINOR: this can be a extracted as toplevel dev for avoid defining it all the time

Reply:
OK